magellan-library/src/main/java/com/wealthfront/magellan/lifecycle/LifecycleAware.kt

```diff
@@ -16,5 +17,7 @@ interface LifecycleAware {
 
   fun destroy(context: Context) {}
 
+  fun onSaveInstanceState(outState: Bundle) {}
```

Comment:
Need this to be passed along to all the children if we wanna save the state across application instances.

Reply:
Not to mention this requires a separate attachment point to the Activity. Also it's useless if we don't pass the bundle to `onCreate`

Reply:
Talked offline. Remove onSaveInstanceState since its not that useful here.